src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -83,28 +83,31 @@ public static class Server {
      * General configuration that doesn't need to be synchronized but needs to be available before server startup
      */
     public static class Common {
+        @Deprecated(since = "1.19.2", forRemoval = true)
         public final BooleanValue cachePackAccess;
+        @Deprecated(since = "1.19.2", forRemoval = true)
         public final BooleanValue indexVanillaPackCachesOnThread;
+        @Deprecated(since = "1.19.2", forRemoval = true)
         public final BooleanValue indexModPackCachesOnThread;
 
         Common(ForgeConfigSpec.Builder builder) {
-            builder.comment("General configuration settings")
+            builder.comment("[DEPRECATED / NO EFFECT]:General configuration settings")
                     .push("general");
 
             cachePackAccess = builder
-                    .comment("Set this to true to cache resource listings in resource and data packs")
+                    .comment("[DEPRECATED / NO EFFECT] [NOW IN BOOT CONFIG]:Set this to true to cache resource listings in resource and data packs")
                     .translation("forge.configgui.cachePackAccess")
                     .worldRestart()
                     .define("cachePackAccess", true);
 
             indexVanillaPackCachesOnThread = builder
-                    .comment("Set this to true to index vanilla resource and data packs on thread")
+                    .comment("[DEPRECATED / NO EFFECT] [NOW IN BOOT CONFIG]:Set this to true to index vanilla resource and data packs on thread")
```

Comment:
```suggestion
                    .comment("[DEPRECATED / NO EFFECT] [NOW IN BOOT CONFIG]: Set this to true to index vanilla resource and data packs on thread")
```
Spacing for readability.